### PR TITLE
Misc minor doc improvements

### PR DESCRIPTION
**Pull request by DDvO**
As motivated by https://github.com/openssl/openssl/pull/17726#discussion_r824341754 and https://github.com/openssl/openssl/pull/17825#issuecomment-1065103035.

### CODE REVIEW

File: doc/man3/OSSL_CMP_CTX_new.pod [hunk 1]
```diff
@@ -449,7 +449,7 @@ The reference counts of those certificates handled successfully are increased.
 OSSL_CMP_CTX_get0_untrusted(OSSL_CMP_CTX *ctx) returns a pointer to the
 list of untrusted certs, which may be empty if unset.
 
-OSSL_CMP_CTX_set1_cert() sets the certificate related to the private key
+OSSL_CMP_CTX_set1_cert() sets the certificate related to the client private key
```
Comment (paulidale) [inline doc/man3/OSSL_CMP_CTX_new.pod:452]: client's private key?

File: doc/man3/OSSL_CMP_CTX_new.pod [hunk 2]
```diff
@@ -477,7 +477,7 @@ Calling this function is optional; by default a chain construction
 is performed on demand that is equivalent to calling this function
 with the I<candidates> and I<own_trusted> arguments being NULL.
 
-OSSL_CMP_CTX_set1_pkey() sets the private key corresponding to the
+OSSL_CMP_CTX_set1_pkey() sets the client private key corresponding to the
```
Comment (paulidale) [inline doc/man3/OSSL_CMP_CTX_new.pod:480]: client's private key?

**Review by paulidale — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Will this apply to 3.0? If so it should be applied there too. @paulidale do you agree?

**Comment by DDvO** [comment]
> Will this apply to 3.0? If so it should be applied there too. @paulidale do you agree?

Yes, this also applies to 3.0, and I just tried: it merges cleanly there, too.

**Comment by DDvO** [comment]
Merged to master and 3.0 - thanks @paulidale and @t8m for the swift reviews.

**Pull request closed without merging**